src/kbmod/filters.py

```diff
@@ -0,0 +1,128 @@
+import abc
+
+from kbmod.result_list import *
+
+
+class Filter(abc.ABC):
```

Comment:
From a code organization angle, I had thought these might live in a subdirectory. As you start to build up filters, this file will get annoyingly big. e.g.

```
src/kbmod/filters/
  __init__.py
  base_filter.py
  lh_filter.py
  num_obs_filter.py
```
And a similar structure to the tests.

Reply:
I think we might want to do that eventually. But let's hold off until we see how many filters we have.